locale/en/recipe-name-bulk.cfg

```diff
@@ -81,11 +81,11 @@ chromite-slagging=Chromite Slagging
 chromate-leaching=Chromate Leaching and Reduction
 chromium-reduction=Chromium Oxide Reduction
 rutile-chlorination=Rutile Chlorination
-titanium-dioxide=Refined Titanium Dioxide
+titanium-oxide=Refined Titanium Dioxide
```

Comment:
OK, titanium-dioxide doesnt exist, but the description should be "Refined Titanium Oxide" then?

Reply:
I place great trust in Repofmes abilities on those matters =)
Also, the chemical formula says TiO2 and wiki mentions dioxide so I think this one's fine:
https://en.wikipedia.org/wiki/Titanium_dioxide

Reply:
It strikes me that you're probably talking about the item definition, not the real world counterpart, when you say titanium-dioxide doesn't exist?
My first pass on this was to rename every instance of titanium-oxide (which is used everywhere else in the code) to titanium-dioxide, but I settled for the much simpler fix. 
I'm not knowledgeable  enough about chemistry and material science to presume corrections to what Repofme has created except for the simple consistency-errors/typos here and there.

Reply:
Yes exactly, I searched the codebase for "titanium-dioxide" and didnt find it. So, my thought was, that the fixed line should be titanium-oxide=Refined Titanium Oxide.
I've not very much knowledge about chemistry. I'm a developer, but dont know much about lua and factorio modding. It was just my sense for consistency, which was triggered :)